hieradata/class/vagrant/mongo.yaml

```diff
@@ -0,0 +1,49 @@
+---
+mongodb::backup::s3_backups: true
+mongodb::server::replicaset_members:
```

Comment:
I was having problem running puppet on a single vagrant mongo-1 instance until I added this. There is an exec that wants to contact all members in the replica set